__tests__/app.test.js

```diff
@@ -86,3 +87,33 @@ describe("GET", () => {
       });
   });
 });
+
+describe("GET", () => {
+  it("should return array of comments when given review_id with following properties", () => {
+    return request(app)
+      .get(`/api/reviews/2/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        expect(comments.length).toBe(3);
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("votes", expect.any(Number));
+          expect(comment).toHaveProperty("created_at", expect.any(String));
+          expect(comment).toHaveProperty("review_id", expect.any(Number));
+          expect(Object.keys(comment)).toHaveLength(6);
+        });
+      });
+  });
+  it("should return comments sorted by created_at in descending order", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
```

Comment:
Lets add some error tests here:
check for status 400, invalid ID, e.g. string of "not-an-id"
check for Status 404, non existent ID, e.g. 0 or 9999